spec/components/provider_interface/application_timeline_component_spec.rb

```diff
@@ -96,6 +96,28 @@ def provider_user
       expect(rendered.css('.govuk-visually-hidden').text).to eq Time.zone.now.to_s(:govuk_date_and_time)
       expect(rendered.css('a').attr('href').value).to eq "/provider/applications/#{application_choice.id}/notes/#{note.id}"
     end
+
+    it 'renders note events by API users' do
+      application_choice = create(:application_choice)
```

Comment:
Consider extracting the applicationchoie to a `let` under the context in line 83.